docs/source/tracking/backend-stores.rst

```diff
@@ -0,0 +1,105 @@
+.. _backend-stores:
+
+==============
+Backend Stores
+==============
+
+The backend store is a core component in `MLflow Tracking <../index.html>`_ where MLflow stores metadata for 
+:ref:`Runs <runs>` and experiments such as:
+
+* Run ID
+* Start & end time
+* Parameters
+* Metrics
+* Code version (only if you launch runs from an :ref:`MLflow Project <projects>`).
+* Source file name (only if you launch runs from an :ref:`MLflow Project <projects>`).
+
+Note that large model artifacts such as model weight files are stored in `artifact store <artifacts-stores.html>`_,  the other core component in MLflow Tracking.
+
+.. _where_runs_are_recorded:
+
+Configure Backend Store
+=======================
+By default, MLflow stores metadata in local files in the ``./mlruns`` directory, but MLflow can store metadata to databases as well.
+You can configure the location by passing **tracking URI** to MLflow, via either of the following methods:
+
+* Set the ``MLFLOW_TRACKING_URI`` environment variable.
+* Call :py:func:`mlflow.set_tracking_uri` in your code.
+* If you are running :ref:`Tracking Server <tracking_server>`, you can set the ``tracking_uri`` option when starting the server, like ``mlflow server --backend-store-uri sqlite:///mydb.sqlite``
+
+Continue to the next section for the supported format of tracking URLs.
+Also visit :ref:`this guidance <tracking_setup>` for how to set up the backend store properly for your workflow.
+
+Supported Store Types
+=====================
+MLflow supports the following types of tracking URI for backend stores:
+
+- Local file path (specified as ``file:/my/local/dir``), where data is just directly stored locally.
+- Database encoded as ``<dialect>+<driver>://<username>:<password>@<host>:<port>/<database>``. MLflow supports the dialects ``mysql``, ``mssql``, ``sqlite``, and ``postgresql``. For more details, see `SQLAlchemy database uri <https://docs.sqlalchemy.org/en/latest/core/engines.html#database-urls>`_.
+- HTTP server (specified as ``https://my-server:5000``), which is a server hosting an :ref:`MLflow tracking server <tracking_server>`.
+- Databricks workspace (specified as ``databricks`` or as ``databricks://<profileName>``, a `Databricks CLI profile <https://github.com/databricks/databricks-cli#installation>`_).
+  Refer to Access the MLflow tracking server from outside Databricks `[AWS] <http://docs.databricks.com/applications/mlflow/access-hosted-tracking-server.html>`_
+  `[Azure] <http://docs.microsoft.com/azure/databricks/applications/mlflow/access-hosted-tracking-server>`_, or :ref:`the quickstart <quickstart_tracking_server>` to
+  easily get started with hosted MLflow on Databricks Community Edition.
+
+.. note::
```

Comment:
We should change this to "important". So many people ask this question. Let's emphasize text to make it stand out as well.

Reply:
Got it! Also worth add it to the FAQ section then.